heltour/tournament/management/commands/cleansedb.py

```diff
@@ -1,21 +1,56 @@
 from django.core.management import BaseCommand
+from heltour.tournament.models import *
+
+from django_comments.models import Comment
+from django.contrib.auth.models import User, Group
+from django.contrib.sessions.models import Session
+from django.contrib.sessions.models import Session
+from reversion.models import Revision
 
 class Command(BaseCommand):
     help = "Cleanse your local database of sensitive things"
 
     def handle(self, *args, **options):
-        from heltour.tournament.models import Player, Registration, FcmSub
-        from django_comments.models import Comment
-        from django.contrib.auth.models import User
+        Session.objects.all().delete()
         for u in User.objects.all():
             u.set_password("09876default1234")
             u.email = "email-{}@example.com".format(u.id)
             u.save()
+            u.user_permissions.all().delete()
+        Group.objects.all().delete()
         for p in Player.objects.all():
+            p.profile = None
```

Comment:
I wouldn't delete the profile. It is all fully public via the lichess API and will break some stuff like blitz ratings.